### ISSUE TITLE
Logo proporsal

### ISSUE THREAD

**Issue by tjulises** [blaa/orgassist#1]
Hi @blaa ,

My name is Ulises or TJ if you like,
i'm a graphic designer and I contribute to cool projects (open source) like this one!
if this is something that you're looking forward to, I'll gladly upload some logo proposals

I'll look forward for your reply, cheers!

TJ.

**Comment by blaa** [comment]
Hello @tjulises!

1) Thank you very much for you offer - I'll gladly include a logo in the README on the main project page, on the PyPI page (I guess it's possible) and if there ever is a web interface it can use it too. ;)


2) Before doing logo I was wondering about changing the project name though. So consider it an open contest for a new name. 
  - "orgassist" is not that bad. It conveys a meaning - assistant which helps you organize, and also connects with org-mode. But it's not that unique:
  - github has (still alive) orgassistant repository.
  - there's orgassist.com.au
  - it should be eventually usable for people without org-mode tree for synchronizing caldav/OWA/google calendar and other notifications into a single stream. But that's ok. "org" has multiple meanings.
  - Currently program entrypoint is "assist.py" - I'm not particularly happy with it either. Didn't make it "orgassist" for technical reasons.
  - I'd like to (subjectively) like the name, and to be simple. Simple names are taken though.

If noone cames up with a better idea in 2 days I believe the topic is closed. Logo doesn't need to relate to the name too, so it might not be a blocker...


3) Please, take into account that the userbase currently is a single person as far as I know. It's usable for me and I'll continue using it and hosting it here. Logo can certainly help popularize the project - but I'd hate for you to work on a project which doesn't really take off. ;)

**Comment by blaa** [comment]
Other ideas I've included in PROJECT.org:
orgally, orgmate, 
orgremote, orgbot (taken, but dead repo?)

Synonyms to assistant: http://www.thesaurus.com/browse/assistant    

those seem taken: botally, botmate

**Comment by tjulises** [comment]
No problem mate. I'll upload my first proposal in this issue, when ready :+1:  @blaa

**Comment by blaa** [comment]
Noone suggested a better name - lets stick with orgassist.

**Comment by tjulises** [comment]
I'd also like to ask some questions (towards the logo-making) @blaa 

What's the name that should be displayed on the logo, If there should be any.

How would you describe this project,
what is it intended to do?

What would you like this logo to transmit?

this little details are going to help me transmit what it's supposed to.

I'll look forward for your reply, cheers!

**Comment by blaa** [comment]
@tjulises:
1) "orgassist" if any. I've got one idea without any letters though (below).

2) It's a "remote control" for org-mode, or an org-mode based "bot-assistant":
a) (org-mode is an note-taking, calendar/agenda-management, mindmap-like software. Related to this project)
b) orgassist helps you manage your calendar take notes on the go, etc. private, text-based and much simpler cousin of Cortana, Alexa, etc. 
c)  Keywords: time organization, meetings, notifications, calendar, notes.

3) It's strictly related to org-mode project which logo you can see here:
http://orgmode.org/img/org-mode-unicorn-logo.png
Let's just say I kind of detest it. Probably because of colors.

But you can relate to this logo. My single idea: a simplified silhouette of a "robo" unicorn (ending with a gear? Edge of a gear with unicorn head? Robo - since "bot", which is a short for robot.

Feel free to do anything really. I like symbolic/simple logos though. A simple expression of letters would be great too.

Cheers,

**Comment by tjulises** [comment]
Ok @blaa I love challenges! I got some ideas already, I'll upload em when I can come up with something good.

**Comment by tjulises** [comment]
Hello! @blaa   this is my first proposal, it can be in any color you want it, it can be adjusted however you want, i'd like to hear any suggestion +1
![unicorn](https://user-images.githubusercontent.com/39039587/41250172-25b5d01e-6d84-11e8-9953-a379d34d1c8c.jpg)

**Comment by blaa** [comment]
Hah! Love them! Though that they are a bit too colorful at the beginning, but meh. It's a unicorn, it's in the package. The more I look the more I like them in fact. I'll give you more heads up tomorrow - and gather some comments. For now I'm mostly between 6 and 7.

Tip of the mane on (6) is kind of undistinguishable without a context - so I'd drop it probably if going for it. 8 and 9 are nice because squared background relates to the computer screen.

Full pro though, thank you - I'll write back tomorrow. ;)

(so far, 3 other people voted 7)

**Comment by therek** [comment]
:+1: for 8

**Comment by blaa** [comment]
So the people are mostly split between 7 and 9, some mentioning 6 and some 1 and 2. 

I believe we can settle on a 7 without additional alterations if you agree. ;-)

So the rest of an answer is just a summary of opinions and my subjective reception to try to give you some useful feedback.

- I liked the winding mechanism at the back so much that I initially dropped 1 to 3. But I like the simplicity and colours of (1) and (2). 
- I guess that the "tip of a mane" problem of 1-6 could be fixed with a contour line on the mane, but that might require some more tweaking... it's hard to visualize. Most people don't notice this as a problem so I might be biased. For me, a silhouette approach can't be just a flattening of a more complicated shape which looses a "spatial information" - If you get me.
- I dropped 4-5 too because of too aggresive colours for my taste. 
- I really liked (6), but I generally like dark themes - in contrast to the most people I know. I even though about using both in cases when there would be light/dark theming involved, but I've got no plans like that for now. 

Going further:
I'm looking forward to hearing you opinion on the matter. ;)

You could make a PR with a final logo to the project (gfx subdir?) or let me do it - as you wish. I'd set a CONTRIBUTORS file and credit you in a README. Would be cool to have both in repo - raster and vector file.

**Comment by tjulises** [comment]
I agree with your decision, number 7 its a good call, your feedback is well received :+1: 

I was thinking on providing the logo in an svg, pdf, png, cdr file on a google drive link and/or I could upload it to source repo, it should be more than necessary and the logo would be ready to use, I'll wait on your feedback! again, glad you liked it :+1: 

Cheers!

**Comment by blaa** [comment]
If svg is completely usable, then I believe svg + png is a great pair. pdf and cdr optionally, but if you believe they add something - please go ahead.

I'd put the files in 'gfx' subdirectory and link README internally to display the logo. You can do this if you like with Github interface - I can squash the GIT history afterwards to keep it clean. Otherwise I can organize the files, but you'd have to provide them first via google drive link for example. ;)

**Comment by tjulises** [comment]
Thanks for letting me be part of the project, it has been a pleasure to be able to collaborate.
here's the [google drive](https://drive.google.com/open?id=1tvfrX8FaE0U9oKKs5EKBVGBpqGL8bKy3) link with all the files versions I said previously, and you can upload it because I'm not ready yet to upload the logo as requested, I'm getting there I've been studying markdown while collaborating to cool projects like this one :+1:  @blaa

**Comment by blaa** [comment]
I've included logo files and contribution note in the README. Please let me know if it's ok; I can change it however you see fit.  Couldn't find you email to give you a more "direct" reference - not sure if you would like it though.

Thank you for the other versions too. (; I'll think of doing something useful with them. If the project gets much more popular, colorful stickers are a must. ;-P
